python/python/tests/tests_server_modules/test_ft.py

```diff
@@ -560,11 +889,7 @@ async def _create_json_keys_for_ft_aggregate_with_bicycles_data(
                 glide_client,
                 prefix + "1",
                 ".",
-                '{"brand": "Bicyk", "model": "Hillcraft", "price": 1200, "description":'
-                + ' "Kids want to ride with as little weight as possible. Especially on an'
-                + ' incline! They may be at the age when a 27.5\\" wheel bike is just too clumsy'
-                + ' coming off a 24\\" bike. The Hillcraft 26 is just the solution they need!",'
-                + ' "condition": "used"}',
+                '{"brand": "Bicyk", "model": "Hillcraft", "price": 1200, "condition": "used"}',
```

Comment:
Why?

Reply:
Upon review of the code across all languages, I found this has already been removed from the JAVA integ tests, I want to keep everything consistent across languages.
The reason for removal from JAVA: It is a key that is not used for anything during testing is adding no value.